### DISCUSSION TITLE
In Simple trigger when the job execution has completed than trigger get deleted from trigger table(QRTZ_TRIGGERS). Can anyone have any idea about how i can resolved this issue

### DISCUSSION THREAD

**Discussion by dinesh1011-cloud** [quartznet/quartznet#2308] [Q&A]
In Simple trigger when the job execution has completed than trigger get deleted from trigger table(QRTZ_TRIGGERS). Can anyone have any idea about how i can resolved this issue

**Comment by xmateosx** [comment]
Maybe you could use a Cron Trigger instead?

**Reply by dinesh1011-cloud** [reply]
I have used Cron also that is working fine but the issue is with simple trigger.
I have used these two different way to create a simple trigger both time it gets deleted from data base
![image](https://github.com/quartznet/quartznet/assets/131234397/93a8b90b-78c8-4b53-9b1e-7872cddeb86a)
![image](https://github.com/quartznet/quartznet/assets/131234397/9cb8cad8-c999-4766-836d-37f00fb7160f)

if you have any idea please let me know

**Comment by jafin** [comment]
@dinesh1011-cloud  I believe this is by design. Once the repeat trigger completes it deletes itself.

```c#
   /// <summary>
    /// Get or set the number of times the <see cref="SimpleTriggerImpl" /> should
    /// repeat, after which it will be automatically deleted.
    /// </summary>
    /// <seealso cref="RepeatIndefinitely" />
    public int RepeatCount
```

Why do you want a completed trigger to persist?

**Reply by dinesh1011-cloud** [reply]
Hi Jafin,

Because our lot of job are dependent on previous jobs status and if we have deleted this trigger from table we are not able to run the next job . Some times we have  triggered these job manually from UI and with latest version every time we have to create new trigger.

earlier  I m using quart.net version(1.0.2.3) where it is not deleting the trigger from table (QRTZ_TRIGGERS) and now we are upgrading our quartz to 3.8.1 where I m facing this issue
